ferdinand-porsche-fern-fachhochschule.csl

```diff
@@ -37,6 +37,10 @@
         <single>et al.</single>
         <multiple>et al.</multiple>
       </term>
+      <term name="in" form="long">
```

Comment:
das kommt mir falsch vor: bei Buchkapiteln (also "chapter") wird doch wohl kaum "Votrag auf der" benutzt werden, oder? (wenn, sollte der term `presented at` und nicht nicht `in` benutzt werden.